.github/workflows/unittests.yml

```
@@ -7,19 +7,21 @@ on:
 
 jobs:
   test:
-    runs-on: ubuntu-18.04
+    runs-on: ubuntu-latest
     container:
       image: ghcr.io/edgelesssys/edgelessrt-dev:ci
 
     steps:
       - name: Checkout
-        uses: actions/checkout@v2
+        uses: actions/checkout@v3
 
       - name: Test
         run: ertgo test -race -count=3 ./...
 
       - name: Setup
-        run: mkdir build
+        run: | 
+          git config --global --add safe.directory "$GITHUB_WORKSPACE"
+          mkdir build
```

```suggestion
        run: mkdir build
```
As far as I can tell the unit test issue was caused by an incompatibility with old git versions and Go 1.18 (we recently updated the used Go version in our CI image), so this should not be needed here

---

Does not appear to be fixed with Go 1.18.

See: 
https://github.com/edgelesssys/marblerun/runs/6213150125 (works with `git config --global --add safe.directory "$GITHUB_WORKSPACE"`)
https://github.com/edgelesssys/marblerun/runs/6213170315 (does not work without it)